docs/GettingStarted.md

```diff
@@ -1386,6 +1378,14 @@ span.context.sampling_priority = Datadog::Ext::Priority::USER_REJECT
 span.context.sampling_priority = Datadog::Ext::Priority::USER_KEEP
 ```
 
+You can disable priority sampling via:
```

Comment:
In `dd-trace-py` we removed this part from our docs, since disabling priority sampling has a chance of breaking distributed tracing, better to just leave this out.

Reply:
Good point.